lib/clientsidescripts.js

```diff
@@ -599,15 +603,36 @@ functions.testForAngular = function(attempts, ng12Hybrid, asyncCallback) {
       asyncCallback(args);
     }, 0);
   };
+  var definitelyNg1 = !!ng12Hybrid;
+  var definitelyNg2 = false;
   var check = function(n) {
     try {
-      if (!ng12Hybrid && window.getAllAngularTestabilities) {
-        callback({ver: 2});
-      } else if (window.angular && window.angular.resumeBootstrap) {
-        callback({ver: 1});
-      } else if (n < 1) {
-        if (window.angular) {
+      // Figure out which version of angular we're waiting on
```

Comment:
These comments should be `/** */` style because some versions of webdriver will glomp lines together and `//` comments break stuff.